### PR TITLE
Add configurable classes to Disclosure

### PR DESCRIPTION
**Pull request by tajakobsen**
Relates to: MT-388

### CODE REVIEW

File: src/svelte/Disclosure.svelte [hunk 1]
```diff
@@ -12,6 +12,10 @@
   export let title: string;
   export let headerTag: 'h2' | 'h3' | 'h4' = 'h3';
   export let theme: 'bordered' | 'links' = 'bordered';
+  export let headerClass: string | undefined;
```
Comment (lha047) [inline src/svelte/Disclosure.svelte:15]: Is there a difference between 
```
export let headerClass: string | undefined;
{panelClass || ''}
```
and
```
export let headerClass: string = '';
{panelClass}
```

Seems easer to read with default value?

**Review by lha047 — CHANGES_REQUESTED** [review]
Good job :D 
Just have a question about using a default value.

File: src/svelte/Disclosure.svelte (hunk 1 above)
Comment (tajakobsen) [reply src/svelte/Disclosure.svelte:15]: I don't have any strong opinions either way, so I can change it.

**Review by lha047 — APPROVED** [review]
Nice :D

**Pull request merged**